layouts/partials/footer.html

```diff
@@ -1,13 +1,6 @@
 <!-- Footer -->
 <footer class="wrapper style1 align-center">
     <div class="inner">
-        <ul class="icons">
```

Comment:
Why are you removing all of this? It worked fine in previous builds of the site that is currently published.

Reply:
@cewood restored them in a32b0c1ca7c9c647d3d802dfa4a42acae9d2ab13 to not block this PR, but IMO these links are broken / invalid / confusing.

Filled https://github.com/ipfs-shipyard/ipfsio-about/issues/12 to discuss